### ISSUE TITLE
How to get transactions per block via RPC?

### ISSUE THREAD

**Issue by cryptoBOOOOM** [cryptonotefoundation/cryptonote#258]

**Comment by SoraKohaku** [comment]
run 2 seed
https://github.com/FndNur1Labs/CryptonoteEvo/wiki/Installation-Base-From-Myself%5BFndNur1Labs%5D

**Comment by cryptoBOOOOM** [comment]
i do not see how this would help...

**Comment by SoraKohaku** [comment]
RPC means for api. just do sync with pool. that rpc thing. and what i know
monero split it for rpc wallet[pool] and cli wallet[general solo].if that
not solve you can ask me telegram for clear solution

**Comment by cryptoBOOOOM** [comment]
yeah i am developing a block explorer and i am trying to get some information via RPC.

I got blockheader, but i need general block information and transactions per block

**Comment by SoraKohaku** [comment]
used api lol...

**Comment by cryptoBOOOOM** [comment]
Okay how can i do this? I synced and got blocks.dat but how do i decode this file?

**Comment by SoraKohaku** [comment]
for here is splitting place...in v2 is rpc_api.hpp

**Comment by Prbcoin** [comment]
Who can help me? I'm creatingWho can help me? I'm testing the currency generated with a pool and I have some problems, you can write to me by telegram @bolivarcash I hope you can help me, thank you.
